src/backend/commands/sequence.c

```diff
@@ -1395,7 +1395,13 @@ init_params(ParseState *pstate, List *options, bool for_identity,
 	/* AS type */
 	if (as_type != NULL)
 	{
-		Oid			newtypid = typenameTypeId(pstate, defGetTypeName(as_type));
+		/*
```

Comment:
Put comment just before hook call and explain that hook might fail to set newtypid in error cases and we will fall back to typenameTypeId. Also use PR template in commit message